styles/style.css

```diff
@@ -0,0 +1,71 @@
+
+
+body {
+    font-family: 'Krona One';font-size: 22px;
+    background-color:#EFEFD0
+}
+h1 {
+ font-style: strong
+
+}
+
+#container {
+  border: 5px solid #ffcc5c;
+  display: flex;
+  flex-direction: column ;
+  justify-content: space-around;
+align-items: center;
+
+}
+
+ul{
+list-style-type: none;
+}
+
+nav ul {
+  display: flex;
+  flex-direction: row  ;
+  justify-content: space-around;
+  background-color:#F7C59F
+}
+
+#button {
```

Comment:
I think the pieces of each CSS block are supposed to be indented over one.

p {
   color:  blue;
}

vs 
p {
color: blue;
}